demo/net.env

```diff
@@ -0,0 +1,17 @@
+# =======================================
+# Network config options (uncomment one)
+# =======================================
+
+# 1. Fast DDS + LAN
+# RMW_IMPLEMENTATION=rmw_fastrtps_cpp
+
+# 2. Cyclone DDS + LAN
+RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
```

Comment:
The cyclonedds doesn't work. The network packages are sending to the different network interface. The configuration to select the network interface in cyclonedds is here https://github.com/husarion/rosbot-xl-autonomy/pull/3 but it would be changed to the dds router.